l10n_ch_hr_payroll/data/hr.salary.rule-new.csv

```diff
@@ -1,28 +0,0 @@
-27,Python Code,True,ALFA_E,Python Expression,,,99999,1,Allocations familiales versées - Enfants à l'école obligatoire,,,result = contract.employee_id.children * 230,result = contract.employee_id.children > 0,1,contract.employee_id.children,51,Free allowance,Allocations familiales
-28,Python Code,True,ALFA_F,Python Expression,,,,,Allocations familiales versées - Enfants en formation,,,result = contract.employee_id.children_student * 300,result = contract.employee_id.children_student > 0,1,contract.wage,51,Free allowance,Allocations familiales
-29,Percentage (%),True,FREP,Always True,,,,,Frais de représentation,5,categories.BASIC, # Available variables: #---------------------- # payslip: object containing the payslips # employee: hr.employee object # contract: hr.contract object # rules: object containing the rules code (previously computed) # categories: object containing the computed salary rule categories (sum of amount of all rules belonging to that category). # worked_days: object containing the computed worked days. # inputs: object containing the computed inputs.  # Note: returned value have to be set in the variable 'result'  result = contract.wage * 0.10, # Available variables: #---------------------- # payslip: object containing the payslips # employee: hr.employee object # contract: hr.contract object # rules: object containing the rules code (previously computed) # categories: object containing the computed salary rule categories (sum of amount of all rules belonging to that category). # worked_days: object containing the computed worked days # inputs: object containing the computed inputs  # Note: returned value have to be set in the variable 'result'  result = rules.NET > categories.NET * 0.10,1,contract.wage,40,Free allowance,
-30,Python Code,True,P13,Always True,,,,,Provision 13ème,,,result = ((categories.GROSS - categories.DED) / 12 ) + categories.COSTS, # Available variables: #---------------------- # payslip: object containing the payslips # employee: hr.employee object # contract: hr.contract object # rules: object containing the rules code (previously computed) # categories: object containing the computed salary rule categories (sum of amount of all rules belonging to that category). # worked_days: object containing the computed worked days # inputs: object containing the computed inputs  # Note: returned value have to be set in the variable 'result'  result = rules.NET > categories.NET * 0.10,1,contract.wage,250,Provision 13th salary,
```

Comment:
After this file is removed, an update of the module on an existing instance will delete all the `hr.salary.rule` above.
Maybe you should keep the same xml ids in the xml file or provide a migration script that remap the xmlids in ir_model_data?